core/src/main/java/nl/nn/adapterframework/processors/InputOutputSenderWrapperProcessor.java

```diff
@@ -60,7 +60,7 @@ public Message sendMessage(SenderWrapperBase senderWrapperBase, Message message,
 				}
 			}
 			if (log.isDebugEnabled()) log.debug(senderWrapperBase.getLogPrefix()+"storing results in session variable ["+senderWrapperBase.getStoreResultInSessionKey()+"]");
-			session.put(senderWrapperBase.getStoreResultInSessionKey(), result.asObject());
+			session.put(senderWrapperBase.getStoreResultInSessionKey(), result);
```

Comment:
Als je dit aanpast, moet alle code die van session variabelen gebruik maakt er tegen kunnen dat het een Message kan zijn.

Reply:
Denk je dat er code is die er niet tegen kan? Ik ben er tot nu toe niet tegenaan gelopen. Als we geen asObject gebruiken dan krijgt Ladybug nl.nn.adapterframework.stream.Message$1@624e2388 binnen voor een stream i.p.v. een Message object waar MessageEncoder aan kan vragen requiresStream(). Moet ik hier iets op verzinnen denk je? Of beter om te proberen alle code om laten gaan met Message object?